manage_breast_screening/record_a_mammogram/tests/test_forms.py

```diff
@@ -1,9 +1,37 @@
 from pytest_django.asserts import assertFormError
+import pytest
 
-from ..forms import ScreeningAppointmentForm
+from manage_breast_screening.clinics.tests.factories import AppointmentFactory
+from ..forms import AppointmentCannotGoAheadForm, ScreeningAppointmentForm
 
 
 class TestScreeningAppointmentForm:
     def test_decision_cannot_be_left_blank(self):
         form = ScreeningAppointmentForm({})
         assertFormError(form, "decision", ["This field is required."])
+
+@pytest.mark.django_db
+class TestAppointmentCannotGoAheadForm:
+    def test_reinvite_reflects_form_data(self):
```

Comment:
It looks like this test is actually 2 tests with a parameter switched. I think this would be a good place to use pytests, parametrize decorator: https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions